tool/jt.rb

```diff
@@ -915,11 +926,11 @@ def test_mri(*args)
 
     files_to_run.sort!
 
-    run_mri_tests(args, files_to_run)
+    run_mri_tests(args, files_to_run, runner_args)
   end
   private :test_mri
 
-  def run_mri_tests(extra_args, test_files, run_options = {})
+  def run_mri_tests(extra_args, test_files, runner_args = [], run_options = {})
```

Comment:
Could you make it a required argument?
Then I think it is more obvious that `run_mri_tests` calls below use no `runner_args`.

Reply:
done